### DISCUSSION TITLE
Ender V3 SE with Nebula screen

### DISCUSSION THREAD

**Discussion by nilrestforthewicken** [Guilouz/Creality-Helper-Script-Wiki#554] [General]
Excuse my ignorance, but would the KE script work with a V3 SE?  I have installed the Nebula screen and camera, and from what I can tell, the Nebula screen is very similar (possibly the same) to the KE screen.  As far as SE vs KE printer hardware, I am not as familiar with their differences.  I did look for a similar discussion, but didn't find one.

**Comment by nilrestforthewicken** [comment]
Is there anyone that could help answer this please?

**Reply by panchomira** [reply]
Its possible, with a rerooted image, https://www.openk1.org/index.php/articles/rooting-the-k1-max-and-ke, check the retail version
